cli/cmd/scan.go

```diff
@@ -72,6 +76,8 @@ func init() {
 		"ID of a defined application to associate the exported vulnerability scan")
 	scanCmd.Flags().BoolP("export", "e", false,
 		"export vulnerability scan results to the backend")
+	scanCmd.Flags().Bool("dockerfile-cis", false,
```

Comment:
maybe `cis-docker-benchmark-scan` , `enables CIS docker benchmark scan. relevant only for image source type`?

Reply:
please also change the flag name `cis-docker-benchmark-scan`